src/database/models/index.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+import fs from 'fs';
+import path from 'path';
+import Sequelize from 'sequelize';
+const basename = path.basename(__filename);
+const env = process.env.NODE_ENV || 'development';
+import configs from '../config';
+const config = configs[env]; 
+const db = {};
+
+let sequelize;
+if (config.use_env_variable) {
+  sequelize = new Sequelize(process.env[config.use_env_variable], config);
+} else {
+  sequelize = new Sequelize(config.database, config.username, config.password, config);
+}
```

Comment:
`if--else block` is not necessary since we have line:8: `const config = configs[env]; ` which is going to return an appropriate database url to use. so consider changing those lines to 
`const sequelize = new Sequelize(process.env[config.use_env_variable], config);`